src/main/java/duke/Command.java

```diff
@@ -0,0 +1,122 @@
+            Messages.printDivider();
+        } catch (IndexOutOfBoundsException e) {
+            Messages.printDivider();
+            System.out.println("No such task exists");
+            Messages.printDivider();
+        } catch (NumberFormatException e) {
+            Messages.printDivider();
+            System.out.println("Invalid command");
+            Messages.printDivider();
+        }
+    }
+
+    public static void executeAdd(ArrayList<String> arguments) {
+        ArrayList<Task> taskList = TaskCollection.getTaskList();
+        try {
+            switch (arguments.get(0)) {
+                case "todo":
+                    taskList.add(new ToDo(arguments.get(1)));
+                    break;
+                case "deadline":
+                    taskList.add(new Deadline(arguments.get(1), arguments.get(2)));
+                    break;
+                case "event":
+                    taskList.add(new Event(arguments.get(1), arguments.get(2)));
+                    break;
+                default:
+            }
+            int taskCount = taskList.size();
+            Messages.printDivider();
+            System.out.println("Got it. I've added this task: ");
+            System.out.println(taskList.get(taskCount - 1));
+            System.out.println("You now have " + taskCount + " items in the list.");
+            Messages.printDivider();
+
+        } catch (IndexOutOfBoundsException e) {
+            Messages.printDivider();
+            System.out.println("Missing parameters");
+            Messages.printDivider();
+        }
+    }
+
+    public static void executeAdd(String input) {
+        String[] arguments = input.split("\\|");
+        ArrayList<Task> taskList = TaskCollection.getTaskList();
+        switch (arguments[0]) {
+            case "T":
+                if(arguments[1].equals("false")) {
+                    taskList.add(new ToDo(arguments[2]));
+                } else {
+                    taskList.add(new ToDo(arguments[2], true));
+                }
+                break;
+            case "D":
+                if(arguments[1].equals("false")) {
+                    taskList.add(new Deadline(arguments[2], arguments[3]));
+                } else {
+                    taskList.add(new Deadline(arguments[2], arguments[3], true));
+                }
+                break;
+            case "E":
+                if(arguments[1].equals("false")) {
+                    taskList.add(new Event(arguments[2], arguments[3]));
+                } else {
+                    taskList.add(new Event(arguments[2], arguments[3], true));
+                }
+                break;
+            default:
+        }
+
+    }
+
+    public static void executeDelete(ArrayList<String> arguments) {
+        try {
+            ArrayList<Task> taskList = TaskCollection.getTaskList();
+            int taskIndex = Integer.parseInt(arguments.get(1)) - 1;
+            taskList.get(taskIndex); //Throw exception early
```

Comment:
Consider writing comments above the code, relative to the position of the code, as per coding standards.